packages/admin-ui/ui/src/domain/orders/details/detail-cards/draft-summary.tsx

```diff
@@ -146,9 +147,9 @@ const DraftSummaryCard: React.FC<DraftSummaryCardProps> = ({ order }) => {
         isFeatureEnabled("inventoryService") &&
         Array.isArray(reservations) && (
           <StatusIndicator
-            onClick={allItemsReserved ? undefined : showAllocationModal}
+            onClick={allItemsReserved ? undefined : showReservationModal}
             variant={allItemsReserved ? "success" : "danger"}
-            title={allItemsReserved ? "Allocated" : "Awaits allocation"}
+            title={allItemsReserved ? "Allocated" : "Not fully reserved"}
```

Comment:
comment: Think we can keep the allocation notion here to have consistent wording in the two possible states:
`Not fully allocated` / `Partially allocated`